dockers/docker-orchagent/ipinip.json.j2

```diff
@@ -2,12 +2,13 @@
 {% set ipv6_addresses = [] %}
 {% set ipv4_loopback_addresses = [] %}
 {% set ipv6_loopback_addresses = [] %}
+{% set loopback_intf_names = ['Loopback0', 'Loopback4096'] %}
```

Comment:
this interface should be in `if multi_asic() `case right?

Reply:
add check to to add Loopback4096 is the device is either Frontend or Backend asic.

Reply:
Just a minor suggestion.. we can have it as below in case if we plan to add more loopbacks in future
```
set loopback_intf_names = ['Loopback0']
if FrontEnd or BackEnd
set loopback_intf_names = loopback_intf_names.append('Loopback4096')
```